Changelog

```diff
@@ -1,9 +1,9 @@
 ## Changelog
 
-- June 04 2023 17:52:23  Giorgio Zoppi <giorgio.zoppi@iotdbe.com>
+- June 04 2024 17:52:23  Giorgio Zoppi <giorgio.zoppi@iotdbe.com>
```

Comment:
I'm guessing this line and the line below got hit by a `2023` => `2024` find and replace all

Reply:
Or giorgio is working from the future